etna/records/views/choosers.py

```diff
@@ -59,17 +58,29 @@ def get(self, request, pk):
             raise Http404
 
 
-class RecordChooserViewSet(ModelChooserViewSet):
+class RecordChooserViewSet(ChooserViewSet):
     """Custom chooser to allow users to filter and select records."""
 
     base_chosen_view_class = KongChosenView
     chooser_mixin_class = KongModelChooserMixinIn
     icon = "form"
-    model = RecordPage
+    model = Record
     page_title = "Choose a record"
     per_page = 10
-    order_by = "iaid"
-    fields = ["iaid", "title"]
+
+    def get_choose_view_attrs(self):
+        attrs = super().get_choose_view_attrs()
+        if hasattr(self, "model"):
```

Comment:
Hi Dan, Do we need to do check if hasattr(self, "model"): ? I can see it has already a model and assigned to the Record

Reply:
Maybe can do  return super().get_choose_view_attrs() ?

Reply:
Or maybe  attrs["model"] = self.model and then return attrs ?